<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>847</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	background:rgb(30, 30, 30);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="styleSheet">
       <string notr="true">QFrame{
      color:rgb(60, 60, 60);
      background:rgb(30,30,30);
      border-top:1px solid darkGray;
      border-bottom:1px solid darkGray;
      border-right:1px solid darkGray;
      border-left:1px solid darkGray;
      border-top-right-radius:5px;
      border-bottom-right-radius:5px; 
      border-top-left-radius:5px;
      border-bottom-left-radius:5px; 
}
QLabel{
	border:none;
    color:white;
    font-size:16px;
    font-weight:700;
	font-family: &quot;微软雅黑&quot;;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color:white;
	background:rgb(50,50,50); ;
    border-radius:3px;
	font-size:15px;
    font-weight:700;
	font-family: &quot;微软雅黑&quot;;
}
QPushButton:hover{
	color:rgb(42,105,192);
    font-size:15px;
    font-weight:700;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="viewManual">
            <property name="text">
             <string>查看使用说明</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame#frame{
      color:rgb(60, 60, 60);
      background:rgb(30,30,30);
      border-top:1px solid darkGray;
      border-bottom:1px solid darkGray;
      border-right:1px solid darkGray;
      border-left:1px solid darkGray;
      border-top-right-radius:5px;
      border-bottom-right-radius:5px; 
      border-top-left-radius:5px;
      border-bottom-left-radius:5px; 
}

QComboBox{
	font-size:12px;
    font-weight:700;
	font-family: &quot;微软雅黑&quot;;
}

QPushButton{
	color:white;
	background:rgb(50,50,50); ;
    border-radius:3px;
	font-size:15px;
    font-weight:700;
	font-family: &quot;微软雅黑&quot;;
}
QPushButton:hover{
	color:rgb(42,105,192);
    font-size:15px;
    font-weight:700;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout" columnstretch="1,4">
            <item row="4" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>-1</pointsize>
                <weight>87</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>操作：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>-1</pointsize>
                <weight>87</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>端口：</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="OpenorClose">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>-1</pointsize>
                <weight>87</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>打开串口</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>-1</pointsize>
                <weight>87</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>波特率：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="baudRate">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>4800</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>115200</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="portId"/>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame#frame{
      color:rgb(60, 60, 60);
      background:rgb(30,30,30);
      border-top:1px solid darkGray;
      border-bottom:1px solid darkGray;
      border-right:1px solid darkGray;
      border-left:1px solid darkGray;
      border-top-right-radius:5px;
      border-bottom-right-radius:5px; 
      border-top-left-radius:5px;
      border-bottom-left-radius:5px; 
}

QLineEdit{
	font-size:12px;
    font-weight:700;
	font-family: &quot;微软雅黑&quot;;
}

QPushButton{
	color:white;
	background:rgb(50,50,50); ;
    border-radius:3px;
	font-size:15px;
    font-weight:700;
	font-family: &quot;微软雅黑&quot;;
}
QPushButton:hover{
	color:rgb(42,105,192);
    font-size:15px;
    font-weight:700;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3" columnstretch="0"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="data_save">
            <property name="text">
             <string>保存文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color:white;
	background:rgb(50,50,50); ;
    border-radius:3px;
	font-size:15px;
    font-weight:700;
	font-family: &quot;微软雅黑&quot;;
}
QPushButton:hover{
	color:rgb(42,105,192);
    font-size:15px;
    font-weight:700;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="handleWidth">
             <number>1</number>
            </property>
            <widget class="QPushButton" name="readDistance">
             <property name="text">
              <string>读取当前距离</string>
             </property>
            </widget>
            <widget class="QPushButton" name="calibrate">
             <property name="text">
              <string>校准基准面</string>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="calculateDepth">
             <property name="text">
              <string>测量深度值</string>
             </property>
            </widget>
            <widget class="QPushButton" name="clearScreen">
             <property name="text">
              <string>清空输出</string>
             </property>
            </widget>
            <widget class="QPushButton" name="quit">
             <property name="text">
              <string>退出程序</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">QLabel{
	border:none;
    color:white;
    font-size:26px;
    font-weight:700;
	font-family: &quot;楷体&quot;;
}
QWidget#customPlot{
	color:#232C51;
    background:rgb(47,48,48);
    border-top:1px solid darkGray;
    border-bottom:1px solid darkGray;
    border-right:1px solid darkGray;
    border-left:1px solid darkGray;
    border-top-right-radius:10px;
    border-bottom-right-radius:10px; 
    border-top-left-radius:10px;
    border-bottom-left-radius:10px;
}
QTextEdit{
    color: white;
	font-family: &quot;JetBrains Mono&quot;;
    background:rgb(75,75,75);
    border-top:1px solid darkGray;
    border-bottom:1px solid darkGray;
    border-right:1px solid darkGray;
    border-left:1px solid darkGray;
    border-top-right-radius:10px;
    border-bottom-right-radius:10px; 
    border-top-left-radius:10px;
    border-bottom-left-radius:10px; 
}
QTextBrowser{
    color: white;
	font-size: 12px;
	font-family: &quot;JetBrains Mono&quot;;
    background:rgb(75,75,75);
    border-top:1px solid darkGray;
    border-bottom:1px solid darkGray;
    border-right:1px solid darkGray;
    border-left:1px solid darkGray;
    border-top-right-radius:10px;
    border-bottom-right-radius:10px; 
    border-top-left-radius:10px;
    border-bottom-left-radius:10px;
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,6,0,3">
       <item row="3" column="0">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item row="3" column="1">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>-1</pointsize>
           <weight>87</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>激光测距图形化界面</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCustomPlot" name="customPlot" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">QLabel{
	border:none;
    color:white;
    font-size:15px;
    font-weight:700;
	font-family: &quot;微软雅黑&quot;;
}</string>
         </property>
         <property name="text">
          <string>基准面距离</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">QLabel{
	border:none;
    color:white;
    font-size:15px;
    font-weight:700;
	font-family: &quot;微软雅黑&quot;;
}</string>
         </property>
         <property name="text">
          <string>深度值</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>:/Images/Icons/gtk-help.png</normaloff>:/Images/Icons/gtk-help.png</iconset>
   </property>
   <property name="text">
    <string>使用说明</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Qt_project/UART_Oscillscope/UART_Oscillscope-master/myresource.qrc"/>
 </resources>
 <connections/>
</ui>
